tools/isoplot/isoplot.xml

```diff
@@ -0,0 +1,155 @@
+    <inputs>
+        <param name='datafile' type='data' format='csv, tabular' multiple='true' label='Upload data file' />
+        <param name='template_file' type='data' format='csv, xlsx' multiple='true' label='Template file' argument='--template'/>
+        <!--param name='run_name' type='text' label='Run name' /-->
+        <param name='format' type='select' label='Plot format'>
+            <option value='pdf'>pdf</option>
+            <option value='jpeg'>jpeg</option>
+            <option value='png'>png</option>
+            <option value='svg'>svg</option>
+            <option value='html'>html</option>
+        </param>
+        <param name='value' type='select' label='Data to plot' >
+            <option value='corrected_area'>Corrected Area</option>
+            <option value='isotopologue_fraction'>Isotopologue Fraction</option>
+            <option value='mean_enrichment'>Mean Enrichment</option>
+        </param>
+        <section name='SP' title='Select static plots to generate'>
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' argument='-bp'
+                   optional='true'/>
+            <param type='boolean' name='generate_meanplot' label='Flag to generate meaned barplots' argument='-mb'
+                   optional='true'/>
+            <!--<param type='boolean' name='generate_areaplot' label='Flag to generate stacked area plots'
+                   argument='-sa' optional='true'/>/-->
+            <param type='boolean' name='generate_heatmap' label='Flag to generate heatmap' argument='-hm'
+                   optional='true'/>
+            <param type='boolean' name='generate_clustermap' label='Flag to generate clustermap' argument='-cm'
+                   optional='true'/>
+        </section>
+        <section name='IP' title='Select interactive plots to generate'>
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' argument='-IB'
+                   optional='true'/>
+            <param type='boolean' name='generate_meanplot' label='Flag to generate meaned barplots' argument='-IM'
+                   optional='true'/>
+           <!-- <param type='boolean' name='generate_areaplot' label='Flag to generate stacked area plots' argument='-IS'
+                   optional='true'/>-->
+            <param type='boolean' name='generate_heatmap' label='Flag to generate heatmap' argument='-HM'
+                   optional='true'/>
+        </section>
+        <param type='boolean' name='stack' label='Flag to unstack bars in barplots' argument='-s' optional='true'/>
+        <param type='boolean' name='verbosity' label='Flag to get debug information' argument='-v' optional='true'/>
+        <param type='boolean' name='annotation' label='Flag to add annotations on the maps' argument='-a'
+               optional='true'/>
+    </inputs>
+    
+    <outputs>
+        <data name='output1' format='csv' label='Formatted data used for plotting' />

```

Comment:
Can we output a tabular file here? Galaxy has a lot of tools for tabular data and this would save a conversion step.
The conversion could happen in a second bash command after the main command.

Reply:
Can you please use more meaningful variable/name names?